File: HouseholdBudgetManager/src/HouseholdBudgetManager.WebApp/projects/household-budget-manager/src/app/models/budget-status.ts
export enum BudgetStatus {
  Draft = 0,
  Active = 1,
  Completed = 2
}

export const BUDGET_STATUS_LABELS: Record<BudgetStatus, string> = {
  [BudgetStatus.Draft]: 'Draft',
  [BudgetStatus.Active]: 'Active',
  [BudgetStatus.Completed]: 'Completed'
};

File: PersonalBudgetTracker/src/PersonalBudgetTracker.Core/Models/BudgetAggregate/Enums/BudgetStatus.cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

namespace PersonalBudgetTracker.Core;

/// <summary>
/// Represents the status of a budget.
/// </summary>
public enum BudgetStatus
{
    /// <summary>
    /// Budget is in draft status.
    /// </summary>
    Draft = 0,

    /// <summary>
    /// Budget is active.
    /// </summary>
    Active = 1,

    /// <summary>
    /// Budget is completed.
    /// </summary>
    Completed = 2,
}
